container/lxd/initialisation_test.go

```diff
@@ -164,6 +164,22 @@ func (s *InitialiserSuite) TestNoSeriesPackages(c *gc.C) {
 	})
 }
 
+func (s *InitialiserSuite) TestInstallViaSnap(c *gc.C) {
+	PatchLXDViaSnap(s, false)
+
+	// Cosmic and beyond use snap as the default package-manager
```

Comment:
"snap as the default package-manager for lxd" ?
I would be surprised if it was the "default" package manager for everything.

Reply:
Whops. Typo from the previous iteration...